http_request/http_request.py

```diff
@@ -105,6 +101,33 @@ def addKeyToObject(self, key="", msg="", data={}):
     def formatJson(self, data):
         return str(data).strip("'<>() ").replace("'", '"')
 
+    def redaction(self, msg):
+        REDACTED = '[REDACTED]'
+
+        msg = str(msg)
+
+        try:
+            # Using MONGO_URI
+            credentials = re.search('(?<=:\/\/).*(?=@)', msg)
+            if credentials:
+                credentials = credentials.group().split(':')
+                msg = msg.replace(credentials[0], REDACTED)
+                msg = msg.replace(credentials[1], REDACTED)
+            
+            #Using MONGO_HOST
+            else: 
+                credentials = re.findall("(?<=\s)'\w+'", msg)
+                if credentials: 
+                    msg = msg.replace(credentials[0], REDACTED)
+                    msg = msg.replace(credentials[1], REDACTED)
+                else:
+                    print("Coudn't find matches")
```

Comment:
we don't need to print anything, it is expected most of the time that error output from backwork will not match your regex's